string robot_name
uint8 command_id
---
bool connected
